API-GUIDELINE.md

```diff
@@ -0,0 +1,282 @@
+# API GUIDELINES
+
+## Overview
+As commercetools is 100% API First, it's important to have a well designed developer experience.
+Additionally we aim to achieve more automation and improve the review process of new changes in the APIs.
+
+With this in mind, we have created "API Guidelines" that aims to help developers to design APIs:
+
+ - simple
+ - consistent 
+ - self-explanatory
+ 
+The Keywords to have a good Guidelines are:
+ 
+ - Consistency => following the Guidelines
+ - Uniformity => checking in the code if there are other fields with similar names and adjusting field naming based on it
+ - Explicability => explaining in a few words the purpose of the field avoiding abbreviations (there are some [exceptions](#exceptions)).
+
+## Scope
+The purpose of these guidelines is to define consistent practices and patterns for all API endpoints.
+
+## URI
+Our URIs are composed by:
+   
+    Base Uri like:
+        https://{service}.{region}.{cloudProvider}.commercetools.com 
+
+    {projectKey}
+
+    resources
+
+    URI parameters like:
+        /{ID}
+        /customer-id={customerId}
+
+
+See our documentation for more details https://docs.commercetools.com/http-api.html#hosts and  https://docs.commercetools.com/http-api-authorization#requesting-an-access-token-using-commercetools-oauth2-server
+
+### Query Parameters
+The Query Parameters can be defined as the optional key-value and not unique that appear after the question mark(?) in the URL and they can be combined with other parameters with an ampersand(&).
```

Comment:
Should be rephrased as it's hard to understand

Reply:
could you please check again? ;)